packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_fr_FR.json

```diff
@@ -14,6 +14,11 @@
   "pci_project_flavors_spec_disk": "Stockage :",
   "pci_project_flavors_spec_bandwidth": "Bande passante :",
   "pci_project_flavors_spec_bandwidth_detail": "{{ bandwidth }} Mbps",
+  "pci_project_flavors_spec_gpu": "Addon :",
+  "pci_project_flavors_spec_gpu_detail_single": "{{ number }} card",
+  "pci_project_flavors_spec_gpu_detail_multi": "{{ number }} cards",
```

Comment:
```suggestion
  "pci_project_flavors_spec_gpu_detail_multi": "{{ number }} cartes",
```

Reply:
This is no longer needed, since the PU was ok re-wording the model to include the number of cards in it